src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -49,13 +49,13 @@ public function __construct(RepositoryInterface $productRepository, SharedStorag
     }
 
     /**
-     * @Given catalog has a product :productName priced at $:price
+     * @Given /^store has a product "([^"]*)" priced at "(€|£|\$)([^"]*)"$/
```

Comment:
Step is configured to match three different currencies (``(€|£|\$)``), so step definition must be a regex and it can't use ``:productName`` as variable.